Qt6 support breaks build on Ubuntu LTS (focal)

### Description

When trying to build master on Ubuntu LTS wit Qt5 cmake fails to run.

```
/tmp$ git clone https://github.com/libfive/libfive.git
Cloning into 'libfive'...
remote: Enumerating objects: 30980, done.
remote: Counting objects: 100% (969/969), done.
remote: Compressing objects: 100% (376/376), done.
remote: Total 30980 (delta 633), reused 855 (delta 573), pack-reused 30011
Receiving objects: 100% (30980/30980), 7.29 MiB | 9.20 MiB/s, done.
Resolving deltas: 100% (21078/21078), done.
/tmp$ cd libfive/
/tmp/libfive$ mkdir build
/tmp/libfive$ cd build/
/tmp/libfive/build$ cmake ..
-- The C compiler identification is GNU 9.4.0
-- The CXX compiler identification is GNU 9.4.0
-- Check for working C compiler: /usr/bin/cc
-- Check for working C compiler: /usr/bin/cc -- works
-- Detecting C compiler ABI info
-- Detecting C compiler ABI info - done
-- Detecting C compile features
-- Detecting C compile features - done
-- Check for working CXX compiler: /usr/bin/c++
-- Check for working CXX compiler: /usr/bin/c++ -- works
-- Detecting CXX compiler ABI info
-- Detecting CXX compiler ABI info - done
-- Detecting CXX compile features
-- Detecting CXX compile features - done
-- Performing Test MARCH_SUPPORTED
-- Performing Test MARCH_SUPPORTED - Success
-- Found Boost: /usr/lib/x86_64-linux-gnu/cmake/Boost-1.71.0/BoostConfig.cmake (found version "1.71.0")  
-- Found PkgConfig: /usr/bin/pkg-config (found version "0.29.1") 
-- Checking for module 'guile-2.2>=2.2.1'
--   No package 'guile-2.2' found
-- Checking for module 'guile-3.0>=3.0.0'
--   Found guile-3.0, version 3.0.1
-- Checking for module 'eigen3>=3.2.92'
--   Found eigen3, version 3.3.7
-- Found ZLIB: /usr/lib/x86_64-linux-gnu/libz.so (found version "1.2.11") 
-- Found PNG: /usr/lib/x86_64-linux-gnu/libpng.so (found version "1.6.37") 
-- Found Python3: /usr/bin/python3.8 (found version "3.8.10") found components: Interpreter Development 
-- Looking for pthread.h
-- Looking for pthread.h - found
-- Performing Test CMAKE_HAVE_LIBC_PTHREAD
-- Performing Test CMAKE_HAVE_LIBC_PTHREAD - Failed
-- Looking for pthread_create in pthreads
-- Looking for pthread_create in pthreads - not found
-- Looking for pthread_create in pthread
-- Looking for pthread_create in pthread - found
-- Found Threads: TRUE  
Checking dependencies:
  libfive:		✓
  Guile bindings:	✓
  Python bindings:	✓
  Studio:		✓   (Python + Guile)
-- Configuring done
CMake Error at studio/CMakeLists.txt:92 (add_executable):
  Target "Studio" links to target "Qt::Widgets" but the target was not found.
  Perhaps a find_package() call is missing for an IMPORTED target, or an
  ALIAS target is missing?


CMake Error at studio/CMakeLists.txt:92 (add_executable):
  Target "Studio" links to target "Qt::Gui" but the target was not found.
  Perhaps a find_package() call is missing for an IMPORTED target, or an
  ALIAS target is missing?


CMake Error at studio/CMakeLists.txt:92 (add_executable):
  Target "Studio" links to target "Qt::OpenGL" but the target was not found.
  Perhaps a find_package() call is missing for an IMPORTED target, or an
  ALIAS target is missing?


CMake Error at studio/CMakeLists.txt:92 (add_executable):
  Target "Studio" links to target "Qt::Concurrent" but the target was not
  found.  Perhaps a find_package() call is missing for an IMPORTED target, or
  an ALIAS target is missing?


-- Generating done
CMake Generate step failed.  Build files cannot be regenerated correctly.

```

### Versions


- Operating system: 
```
/tmp$ cat /etc/lsb-release 
DISTRIB_ID=Ubuntu
DISTRIB_RELEASE=20.04
DISTRIB_CODENAME=focal
DISTRIB_DESCRIPTION="Ubuntu 20.04.4 LTS"
```
- `libfive` commit hash: 
```
$ git rev-parse HEAD
03b592cfc0fa3d9b3cf8117c191e0836d88a89a3
```
  - Is this the latest commit to `master`?  yes
- Qt version (if applicable): 
```
$ dpkg --get-selections | grep libqt
libqt53danimation5:amd64			install
libqt53dcore5:amd64				install
libqt53dextras5:amd64				install
libqt53dinput5:amd64				install
libqt53dlogic5:amd64				install
libqt53dquickscene2d5:amd64			install
libqt53drender5:amd64				install
libqt5concurrent5:amd64				install
libqt5core5a:amd64				install
libqt5dbus5:amd64				install
libqt5designer5:amd64				install
libqt5designercomponents5:amd64			install
libqt5gui5:amd64				install
libqt5help5:amd64				install
libqt5keychain1:amd64				install
libqt5network5:amd64				install
libqt5opengl5:amd64				install
libqt5opengl5-dev:amd64				install
libqt5positioning5:amd64			install
libqt5printsupport5:amd64			install
libqt5qml5:amd64				install
libqt5quick5:amd64				install
libqt5quicktest5:amd64				install
libqt5quickwidgets5:amd64			install
libqt5script5:amd64				install
libqt5sensors5:amd64				install
libqt5serialport5:amd64				install
libqt5sql5:amd64				install
libqt5sql5-sqlite:amd64				install
libqt5svg5:amd64				install
libqt5test5:amd64				install
libqt5webchannel5:amd64				install
libqt5webkit5:amd64				install
libqt5widgets5:amd64				install
libqt5x11extras5:amd64				install
libqt5xml5:amd64				install
libqt5xmlpatterns5:amd64			install
```

```
### Additional Information

Building before the Qt6 support branch works:
```
$ git rev-parse HEAD
c36ac1e5a8713bc0c970982cc587f6a498278e41
```
(cmake and make finish without errors)

--- a/studio/CMakeLists.txt
+++ b/studio/CMakeLists.txt
@@ -100,11 +100,12 @@ target_include_directories(Studio PUBLIC include)
 target_link_libraries(Studio libfive libfive-stdlib ${EXTRA_LIBS})
 
 if (Qt5Core_FOUND)
-    target_link_libraries(Studio
-        Qt::Widgets
-        Qt::Gui
-        Qt::OpenGL
-        Qt::Concurrent
+    target_link_libraries(Studio libfive libfive-stdlib
+        Qt5::Widgets
+        Qt5::Gui
+        Qt5::OpenGL
+        Qt5::Concurrent
+        ${EXTRA_LIBS}
     )
 elseif (Qt6Core_FOUND)
     target_link_libraries(Studio

